phpcs.xml.dist

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<ruleset>
+    <arg name="basepath" value="."/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="80"/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+
+     <!-- Ignore warnings, show progress of the run and show sniff names -->
+    <arg value="nps"/>
+
+    <rule ref="Doctrine"/>
+
+    <file>lib/</file>
+    <file>tests/</file>
+
+    <exclude-pattern>lib/Doctrine/Annotations/Annotation/*</exclude-pattern>
```

Comment:
Lots and lots of excludes: is this worth doing in first place, if half the lib isn't covered at all?

Reply:
It's for the new files we add, e.g. #247, new parser etc.